code/modules/power/lighting.dm

```diff
@@ -32,12 +32,22 @@
 	var/sheets_refunded = 2
 	/// Holder for the completed fixture
 	var/obj/machinery/light/newlight = null
+	/// Power cell for emergency lighting
+	var/obj/item/stock_parts/cell/cell
```

Comment:
Can't say I'm a fan of storing a power cell in the frame item. I'd rather it gets placed in the light structure directly.